server/src/routes/interviews.ts

```diff
@@ -18,4 +19,22 @@ router.get("/candidate/:id", auth, listInterviewsCandidate);
 router.get("/hr/:id", auth, listInterviewsHR);
 router.get("/:id", getMeeting);
 
+router.get("/:id/endtime", async (req, res) => {
```

Comment:
move the logic in controllers

Reply:
Opened an issue for this improvement. Merging this PR.